playbooks/aws/openshift-minion/launch.yml

```diff
@@ -65,5 +70,11 @@
       delay: 10
       with_items: ec2.instances
 
-# Apply the configs, seprate so that just the configs can be run by themselves
-- include: config.yml
+- name: "Configure instances"
```

Comment:
I see, but we cannot configure the master before having created the minions boxes because the master needs to know the IPs of the minions (`--nodes` parameter in /etc/sysconfig/openshift)

A solution could be to merge the two launch.yml files and to merge the two config.yml files.
What do you think?